sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormRecognizerClient.cs

```diff
@@ -168,19 +190,28 @@ public virtual async Task<RecognizeContentOperation> StartRecognizeContentAsync(
         /// <param name="cancellationToken">A <see cref="CancellationToken"/> controlling the request lifetime.</param>
         /// <returns>A <see cref="RecognizeContentOperation"/> to wait on this long-running operation.  Its <see cref="RecognizeContentOperation.Value"/> upon successful
         /// completion will contain layout elements extracted from the form.</returns>
-        [ForwardsClientCalls]
         public virtual RecognizeContentOperation StartRecognizeContentFromUri(Uri formUri, RecognizeOptions recognizeOptions = default, CancellationToken cancellationToken = default)
         {
             Argument.AssertNotNull(formUri, nameof(formUri));
 
-            SourcePath_internal sourcePath = new SourcePath_internal()
+            recognizeOptions ??= new RecognizeOptions();
```

Comment:
I just realized we are not using this property for this method.... should we just remove it?

Reply:
IncludeFieldElements is on by default for this API and ContentType doesn't make sense for URL

Reply:
We realized that in this PR as well. I think that removing it is a valid approach, but I'm not 100% sure if it's worth it. We expect the `RecognizeOptions` type to expand with future service updates (that was the whole point of using a property bag even though we have only two options). We could remove it for the `StartRecognizeContentFromUri` methods only, but it's likely that we'll need overloads that take a property bag in the future. I added:
```cs
recognizeOptions ??= new RecognizeOptions();
```
here because, even though we're not using the options now, I'm pretty sure that we would forget to add this line of code once we started using them in this method. It's more about keeping the same pattern everywhere to avoid mistakes in the future. We have a petty perf cost here, but that's it.

Regarding your original question, I don't feel strongly about keeping/removing the options parameter here. We can consult with the architects if we feel that it's a concern, though.

Related: having unused options (such as `ContentType` when calling any `FromUri` methods) seems a bit off to me, but a redesign may not be worth at this point.

Reply:
I don't feel strongly about it either. It might be confusing now, but in the future it might be useful.
I also like the consistency with other methods.

Thanks for the discussion!